<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #141414;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_registr">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: red;
	font-size: 20px;
}

QPushButton:hover {
    color: white;
}</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="registr_login">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 16px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="second_password">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 16px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="first_password">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 16px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 12px;</string>
    </property>
    <property name="text">
     <string>Логин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 12px;</string>
    </property>
    <property name="text">
     <string>Повторите пароль:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 12px;</string>
    </property>
    <property name="text">
     <string>Пароль:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
